holdem/env.py

```diff
@@ -431,7 +426,7 @@ def _output_state(self, current_player):
       'pocket_cards': current_player.hand,
       'pot': self._totalpot,
       'button': self._button,
-      'tocall': (self._tocall - current_player.currentbet),
+      'tocall': (self._tocall),
```

Comment:
These parentheses are not required.

Reply:
Just a leftover from the original version.

Reply:
Removed